Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -1103,6 +1118,12 @@ function vtkOpenGLTexture(publicAPI, model) {
       newArray
     );
 
+    model.context.finish();
+    const err = model.context.getError();
+    if (err !== model.context.NO_ERROR) {
+      vtkErrorMacro('bad stuff');
```

Comment:
Is this debugging code or is this meant to be here?

Reply:
not meant to be there